makedefs checks whether the test binary it builds can be run

diff -wpruN --no-dereference '--exclude=*.orig' a~/makedefs a/makedefs
--- a~/makedefs	1970-01-01 00:00:00
+++ a/makedefs	1970-01-01 00:00:00
@@ -877,6 +877,7 @@ case "$CCARGS" in
 		  (icu-config --cppflags)) 2>/dev/null` && {
 		icu_ldflags=`((pkg-config --libs icu-uc icu-i18n) ||
 		  (icu-config --ldflags)) 2>/dev/null` && {
+		    icu_ldflags+=" -Wl,-R/usr/lib/$(MACH64)"
 		    trap 'rm -f makedefs.test makedefs.test.[co]' 1 2 3 15
 		    cat >makedefs.test.c <<'EOF'
 #include <unicode/uidna.h>
